src/actinia_core/models/process_chain.py

```diff
@@ -266,6 +266,34 @@ class OutputParameter(IOParameterBase):
                     "dbstring": "PG:host=localhost dbname=postgis user=postgres",
                     "output_layer": "roads"}
     }
+    properties["metadata"] = {
+        'type': 'object',
+        'properties': {
+            'format': {
+                'type': 'string',
+                'description': 'Format of the metadata file. Only STAC is supported'
+                               'The STAC item builder works just on raster export file.'
+                               'These files are accesible through a STAC Catalog'
+                               'and each export is stored as STAC Item',
+                'enum': ['STAC']
+            },
+            'type': {
+                'type': 'string',
+                'description': 'The type of the output. In this case'
+                               'metadata is the only option currently supported',
+                'enum': ['metadata']
+            },
+            'output_layer': {
+                'type': 'string',
+                'description': 'Prefix for STAC item name and ID. REQUIRED'
```

Comment:
Is it really required? I thought the previous comment was more clear in that it uses the output map name if not set - at least this is what I was expecting. Just the word `vector` didn't fit too well as all types of output can be used.

Reply:
Yes, I think we defined it as the attribute to give the name to the `STRDS`